subjects/devops/greatest-of-all/README.md

```diff
@@ -19,9 +19,21 @@ Enter a number: 24
 Enter a number: 45
 Enter a number: 2
 The largest number is 45
+$ ./greatest-of-all.sh
```

Comment:
I missed this in the previous review, on line 11 I assume what you meant is `by` instead of `either`?